src/components/Property/Types/NumericProperty/NumericProperty.tsx

```diff
@@ -0,0 +1,84 @@
+import { Flex, Group } from '@mantine/core';
+
+import { usePropListeningState } from '@/api/hooks';
+import { NumericInput } from '@/components/Input/NumericInput/NumericInput';
+import { PropertyLabel } from '@/components/Property/PropertyLabel';
+
+import { NumericPropertySlider } from './NumericPropertySlider';
+
+interface Props {
+  name: string;
+  description: string;
+  disabled: boolean;
+  setPropertyValue: (newValue: number) => void;
+  value: number;
+  additionalData: {
+    Exponent: number;
+    MaximumValue: number;
+    MinimumValue: number;
+    SteppingValue: number;
+  };
+  // TODO: view options in metadata
+}
```

Comment:
Can we extend Props from a Property type?

Reply:
I would have to create a new type, but yes, it is possible. 

However, the only things I can really put in that type are `name`, `description`, and `disabled`. All the other arguments will be specific to the given property type

Reply:
Would be good to discuss ideas for the property handling, if the current approach is confusing.